app/src/components/TopBar/TopBar.tsx

```diff
@@ -25,7 +25,7 @@ const TopBar: React.FC = () => {
           justifyContent="flex-start"
           display="flex"
           flexDirection="row"
-          height={65}
+          height={75}
         >
```

Comment:
why 75? I like 72

Reply:
75 seems to be the most centered, but I could be wrong.  as long as its >70 it looks fine